test/e2e/receive_test.go

```diff
@@ -70,35 +70,33 @@ func TestReceive(t *testing.T) {
 		// receiver in the hashring than the one handling the request.
 		// The querier queries all the receivers and the test verifies
 		// the time series are forwarded to the correct receive node.
-		r1, err := e2ethanos.NewReceiver(s.SharedDir(), s.NetworkName(), "1", 1)
+		r1, err := e2ethanos.NewReceiver(s.SharedDir(), s.NetworkName(), "1")
 		testutil.Ok(t, err)
-		r2, err := e2ethanos.NewReceiver(s.SharedDir(), s.NetworkName(), "2", 1)
+		r2, err := e2ethanos.NewReceiver(s.SharedDir(), s.NetworkName(), "2")
 		testutil.Ok(t, err)
-		r3, err := e2ethanos.NewReceiver(s.SharedDir(), s.NetworkName(), "3", 1)
+		r3, err := e2ethanos.NewReceiver(s.SharedDir(), s.NetworkName(), "3")
 		testutil.Ok(t, err)
 
-		h := receive.HashringConfig{
+		h := route.HashringConfig{
 			Endpoints: []string{
 				r1.GRPCNetworkEndpointFor(s.NetworkName()),
 				r2.GRPCNetworkEndpointFor(s.NetworkName()),
 				r3.GRPCNetworkEndpointFor(s.NetworkName()),
 			},
 		}
 
-		// Recreate again, but with hashring config.
-		r1, err = e2ethanos.NewReceiver(s.SharedDir(), s.NetworkName(), "1", 1, h)
-		testutil.Ok(t, err)
-		r2, err = e2ethanos.NewReceiver(s.SharedDir(), s.NetworkName(), "2", 1, h)
+		// Create a router.
+		route1, err := e2ethanos.NewRouter(s.SharedDir(), s.NetworkName(), "1", 1, h)
 		testutil.Ok(t, err)
-		r3, err = e2ethanos.NewReceiver(s.SharedDir(), s.NetworkName(), "3", 1, h)
-		testutil.Ok(t, err)
-		testutil.Ok(t, s.StartAndWaitReady(r1, r2, r3))
+		x := s.StartAndWaitReady(r1, r2, r3, route1)
+		println(x)
+		time.Sleep(5 * time.Minute)
```

Comment:
Why do we need to sleep here?

Reply:
Seems like this was written to debug, removing this